pkg/certs/cert-inspection/certgraphapi/type_registry.go

```diff
@@ -0,0 +1,45 @@
+package certgraphapi
+
+// PKIRegistryInfo holds information about TLS artifacts stored in etcd. This includes object location and metadata based on object annotations
+type PKIRegistryInfo struct {
+	// +mapType:=atomic
+	CertificateAuthorityBundles []PKIRegistryInClusterCABundle `json:"certificateAuthorityBundles"`
```

Comment:
I'll allow a followup, but I suspect this object grows into `inClusterCertificateAuthorityBundles` on `onDiskCertificateAuthorityBundles`

Reply:
I'd rather have `PKIRegistryInfo` include `OnDiskPKIRegistryInfo` and `ClusterPKIRegistryInfo` structs, but yeah, we could merge them in one structure later